pkg/inject/report.go

```diff
@@ -154,6 +158,14 @@ func (r *Report) disableByAnnotation(conf *ResourceConfig) (bool, string, string
 		return podAnnotation == k8s.ProxyInjectDisabled, "", ""
 	}
 
+	if !isInjectAnnotationValid(conf.nsAnnotations) {
```

Comment:
Instead of adding this helper function to iterate through the annotations, can we check the `podAnnotation` and `nsAnnotation` variables from line [154-155](https://github.com/linkerd/linkerd2/blob/93db0578fbff7cd61a0f924b19268d8bfa9fcfeb/pkg/inject/report.go#L154-L155) to make sure if they are non-empty, then they must be either `enabled` or `disabled`?